src/core/server/extensions/playerFunctions/sync.ts

```diff
@@ -292,7 +292,7 @@ const Sync = {
 
     playTime(playerRef: alt.Player): void {
         const player = playerRef as ReadOnlyPlayer;
-        Athena.state.set(player, 'hours', player.data.hours === 0 ? 0 : player.data.hours + 0.0166666666666667, true);
+        Athena.state.set(player, 'hours', player.data.hours + 0.0166666666666667, true);
```

Comment:
This will throw an exception when hours is not set or the character is not loggedin (yet?)

Reply:
Yes, but you never know what other devs using this framework do. They may modify core code breaking stuff and maybe even remove hours entirely. Basically we're in undefined behaviour land hence the request.

Reply:
Hence why i suggested the following snippet to Lord privately (i think):
```ts
Athena.state.set(player, 'hours', (player.data?.hours ?? 0) + 0.0166666666666667, true);
```

Reply:
You can add through commiting again on your feature-branch or 3.7.0 what xLuxy has stated, it will be more safe in future cases, so just add another commit to the branch where you have pull-requested from.

There can be alot of cases where someone might want to remove it i guess?